duckduckgo/iOS
# Add Checkmark icon to pickers

Task/Issue URL: https://app.asana.com/0/0/1206327436431895/f

**Description**:
- Adds a checkmark icon to the picker options in settings (based on design feedback)
- Fixes missing translations

<img width="546" alt="Screenshot 2024-01-12 at 7 17 48 PM" src="https://github.com/duckduckgo/iOS/assets/1156669/ac0a2dfe-a436-4bee-9841-8e55a9f9b922">

**Steps to test this PR**:
1. Go to Settings
2. Tap any option that displays a picker (Theme, Fire Button Animation or Address bar Position)
3. Observe there's a checkmark icon in the selected option
4. Test with iOS 15.x and observe menus for the above options work
5. Change device language 
6. Check settings main view is properly translated



---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
